citest/service_testing/agent_test_case.py

```diff
@@ -540,7 +542,7 @@ def run_test_case(self, test_case, timeout_ok=False,
         self.logger.debug('Exception was at:\n%s', traceback.format_exc())
       except BaseException as unexpected:
         self.logger.error(
-            'Unexpected error {0}\nHandling original exception {1}',
+            'Unexpected error %s\nHandling original exception %s',
```

Comment:
I'm using positional parameters to the logger. I think {0} is only for format strings. Or at least I was getting a lint error here.